script.sql

```diff
@@ -0,0 +1,13 @@
+select * from student
+where age between '11' and '15'
```

Comment:
```suggestion
where age between 11 and 15
```
возраст - это числа, а не строки